deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/ModeShapeSubsystemXMLReader_3_0.java

```diff
@@ -31,7 +31,7 @@
 import org.jboss.staxmapper.XMLElementReader;
 import org.jboss.staxmapper.XMLExtendedStreamReader;
 
-public class ModeShapeSubsystemXMLReader_2_1 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
+public class ModeShapeSubsystemXMLReader_3_0 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
```

Comment:
No it would not, but the only required change would be:
```
 <subsystem xmlns="urn:jboss:domain:modeshape:3.0">
```
So only the namespace number has to be changed, which is more than reasonable IMO. It reflects the change of major version, which will translate into a whole bunch of configuration changes.